grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -46,14 +46,17 @@ class OrderController {
         // Set default values
         params.destination = params.destination?:session?.warehouse?.id
         params.orderTypeCode = params.orderTypeCode ? Enum.valueOf(OrderTypeCode.class, params.orderTypeCode) : OrderTypeCode.PURCHASE_ORDER
-        params.status = params.status ? Enum.valueOf(OrderStatus.class, params.status) : null
+        params.status = null
 
         // Pagination parameters
         params.max = params.format ? null : params.max?:10
         params.offset = params.format ? null : params.offset?:0
 
         def orderTemplate = new Order(params)
         def orders = orderService.getOrders(orderTemplate, statusStartDate, statusEndDate, params)
+        if (params.displayStatus) {
```

Comment:
What about pagination? What happens when you paginate the results in the getOrders call and then filter afterwards.